conda_build/metadata.py

```diff
@@ -44,6 +44,9 @@
 ARCH_MAP = {'32': 'x86',
             '64': 'x86_64'}
 
+output_re = re.compile(r"^\s+-\s(?:name|type):.+?(?=^\w|\Z|^\s+-\s(?:name|type))",
```

Comment:
This is used to match all outputs, each as a single match of this re.  We then index by numeric order according to name matching of rendered outputs section.